CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Added
 
+- [[#275](https://github.com/rust-vmm/kvm-ioctls/pull/275)]: Introduce `riscv64` ioctls.
```

Comment:
Should the README's platform list also be updated to reflect this?

Reply:
Thanks for reminding! I'll put it into `READMD` too 😊